src/utils/update/mapper.utils.ts

```diff
@@ -3,6 +3,7 @@ import { OfficeAddress, ServiceAddress } from "@companieshouse/api-sdk-node/dist
 import { Address as PSCAddress, DateOfBirth as PSCDateOfBirth, NameElements } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
 import { Address as OfficerAddress, DateOfBirth as OfficerDateOfBirth } from "@companieshouse/api-sdk-node/dist/services/company-officers/types";
 import { padWithZero } from "../../controllers/update/update.review.beneficial.owner.individual";
+import { privateADDRESS } from "@companieshouse/api-sdk-node/dist/services/overseas-entities";
```

Comment:
I believe this has been updated to the following
```suggestion
import { privateAddress } from "@companieshouse/api-sdk-node/dist/services/overseas-entities";
```